examples/experiments/src/websocket.ts

```diff
@@ -0,0 +1,29 @@
+import { createWebSocket } from "@solidjs/start/websocket";
+
+export default createWebSocket({
+  upgrade(req) {
+    console.log(`[ws] upgrading ${req.url}...`)
+    return {
+      headers: {}
+    }
+  }
```

Comment:
```suggestion
  },
```
think there's a missing comma here!